app/main/views/suppliers.py

```diff
@@ -343,6 +343,9 @@ def submit_company_summary():
 
 @main.route('/create-your-account', methods=['GET'])
 def create_your_account():
+    current_app.logger.info(
+        "suppliercreate: get create-your-account supplier_id:{}".format(
+            session['email_supplier_id']))
```

Comment:
Might be worth doing a `.get` with a default here, just in case (I know people shouldn't be going direct to the URL without a session in place, but you never know...)